src/Symfony/Component/Notifier/Bridge/Lox24/Lox24Transport.php

```diff
@@ -0,0 +1,187 @@
+    }
+
+    public function supports(MessageInterface $message): bool
+    {
+        return $message instanceof SmsMessage
+            && (null === $message->getOptions() || $message->getOptions() instanceof Lox24Options);
+    }
+
+    /**
+     * @throws RedirectionExceptionInterface
+     * @throws DecodingExceptionInterface
+     * @throws ClientExceptionInterface
+     * @throws TransportExceptionInterface
+     * @throws ServerExceptionInterface
+     */
+    protected function doSend(MessageInterface $message): SentMessage
+    {
+        if (!$message instanceof SmsMessage) {
+            throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
+        }
+
+        $from = $message->getFrom() ?: $this->from;
+
+        if (!$this->isFromValid($from)) {
+            throw new InvalidArgumentException(sprintf('The "From" number "%s" is not a valid phone number, shortcode, or alphanumeric sender ID.', $from));
+        }
+
+        $body = [
+            'sender_id' => $from,
+            'phone' => $message->getPhone(),
+            'text' => $message->getSubject(),
+        ];
+
+        $options = $message->getOptions()?->toArray() ?? [];
+        $body = $this->setIsTextDeleted($body, $options);
+        $body = $this->setCallbackData($body, $options);
+        $body = $this->setDeliveryAt($body, $options);
+        $body = $this->setServiceCode($body, $options);
+        $body = $this->setVoiceLang($body, $options);
+
+        $response = $this->client->request('POST', sprintf('https://%s/sms', $this->getEndpoint()), [
+            'headers' => [
+                'X-LOX24-AUTH-TOKEN' => sprintf('%s:%s', $this->user, $this->token),
+                'Accept' => 'application/json',
+                'Content-Type' => 'application/json',
+                'User-Agent' => 'LOX24 Symfony Notifier',
+            ],
+            'body' => $body,
+        ]);
+
+        try {
+            $statusCode = $response->getStatusCode();
+        } catch (TransportExceptionInterface $e) {
+            throw new TransportException('Could not reach the remote LOX24 server.', $response, 0, $e);
+        }
+
+        if (201 !== $statusCode) {
+            $error = $response->toArray(false);
+
+            throw new TransportException(sprintf('Unable to send the SMS: "%s".', $error['detail']), $response);
+        }
+
+        $success = $response->toArray(false);
+
+        $sentMessage = new SentMessage($message, (string) $this);
+        $sentMessage->setMessageId($success['uuid']);
+
+        return $sentMessage;
+    }
+
+    private function isFromValid(string $from): bool
+    {
+        return preg_match('/^[.\-a-zA-Z0-9_ ]{2,11}$/', $from) || preg_match('/^\+[1-9]\d{1,14}$/', $from);
+    }
+
+    private function setIsTextDeleted(array $body, array $options): array
+    {
+        $body['is_text_deleted'] = (bool) ($options['is_text_delete'] ?? false);
```

Comment:
is_text_deleteD or without the D? Just want to be sure

Reply:
is_text_deleteD